src/map/zone.cpp

```diff
@@ -244,6 +244,38 @@ void CZone::SetBackgroundMusicNight(uint8 music)
     m_zoneMusic.m_songNight = music;
 }
 
+const QueryByNameResult_t& CZone::queryEntitiesByName(std::string const& name)
+{
+    TracyZoneScoped;
+
+    // Use memoization since lookups are typically for the same mob names
+    auto result = m_queryByNameResults.find(name);
+    if (result != m_queryByNameResults.end())
+    {
+        return result->second;
+    }
+
+    std::vector<CBaseEntity*> entities;
+
+    // TODO: Make work for instances
+    ForEachNpc([&](CNpcEntity* PNpc)
```

Comment:
Minor formatting here (clang-format always seems to trip up on lambdas)

Reply:
What's the formatting issue? I now know why there was a clang-format off above the lambdas.

Reply:
bracing/alignment
https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md#lambdas

Reply:
Alright, I believe that is the correct formatting

Reply:
The change I made look correct to me according to the second example in contributing; however, the sanity checker is complaining that I need to change it back to what I had before. Is this a discrepancy between clang formatter and contributing guidelines or am I missing something?

Reply:
looks correct to me now, I think ci end is making the mistake